src/backends/torch/torchlib.cc

```diff
@@ -399,13 +427,33 @@ namespace dd
   void TorchModule::load(TorchModel &model)
   {
     if (!model._traced.empty() && model._proto.empty())
-      _traced = std::make_shared<torch::jit::script::Module>(
-          torch::jit::load(model._traced, _device));
+      {
```

Comment:
Maybe put this block into a function, e.g. `traced_model_load()` ?

Reply:
ok